common/app/templates/headerInlineJS/config.scala.js

```diff
@@ -18,7 +18,8 @@ window.guardian = {
             )
     ),
     css: {
-        loaded: false
+        loaded: false,
+        loadedListeners: []
```

Comment:
I'd call this `loadQueue` or something similar, since they're not listening, they're called